README.md

```diff
@@ -1,41 +1,6 @@
 uData
 =====
 
-[![Build status][travis-badge]][travis-url]
-[![Code coverage][coveralls-badge]][coveralls-url]
-[![Python Requirements Status][requires-io-badge]][requires-io-url]
-[![JavaScript Dependencies Status][david-dm-badge]][david-dm-url]
-[![JavaScript Development Dependencies Status][david-dm-dev-badge]][david-dm-dev-url]
-[![Join the chat at https://gitter.im/etalab/udata][gitter-badge]][gitter-url]
```

Comment:
For me, check quickly the dependencies and latest build
For others, find quickly the gitter chan
Coverage can be removed as we don't use it anymore